--- qcontrol-0.5.4/Makefile	2014-04-11 18:03:50.000000000 +0200
+++ qcontrol-0.5.4-neu/Makefile	2014-04-23 16:51:43.118452000 +0200
@@ -1,14 +1,13 @@
 VERSION=0.5.4
 
-CFLAGS   += -c -g -Os -Wall -Wextra
+CFLAGS   += -c
 CPPFLAGS += -DQCONTROL_VERSION=\"$(VERSION)\"
 
-LDFLAGS  += -g
 LIBS     += -lpthread
 LIBS_STATIC += /usr/lib/liblua5.1.a -lpthread -lm -ldl
 
-CFLAGS   += $(shell pkg-config --cflags lua5.1)
-LIBS     += $(shell pkg-config --libs lua5.1)
+CFLAGS   += $(shell pkg-config --cflags lua)
+LIBS     += $(shell pkg-config --libs lua)
 
 ifeq ($(shell pkg-config --exists libsystemd-daemon 2>/dev/null && echo 1),1)
 CPPFLAGS += -DHAVE_SYSTEMD
